stage1/modules/sass/en/README.md

```diff
@@ -0,0 +1,25 @@
+# [Stage#1](../../../) CSS Preprocessors. Sass
+
+## In this module, students need to:
+
+Know and understand:
+
+- types and tasks of preprocessors
+- SASS basics (variables, inheritance,
+  selectors, operators)
+- imports, mixins, comments
+- placeholders, functions, media queries
+
+## Approximate time to complete this module:
+
+3 hours
+
+## Theory
+
+1. [Preprocessors. SASS](https://www.youtube.com/watch?v=JO8DvVZbxDw&feature=youtu.be) - 2 hours
+   - [Slides (in Russian)](https://slides.com/viktoryiavorozhun/deck)
+2. [Sass Basics](https://sass-lang.com/guide/) - 1 hour
```

Comment:
Please add more links to the materials.
https://sherocommerce.com/what-is-a-css-preprocessors-why-use-them/
https://postcss.org/